wandb/sdk/wandb_run.py

```diff
@@ -567,6 +573,23 @@ def _history_callback(self, row=None, step=None):
                 }
                 row[k] = row[k].value
                 visualize_persist_config = True
+            elif isinstance(row[k], CustomChart):
+                custom_charts[k] = row[k]
+                custom_chart = row[k]
+
+        for k, custom_chart in custom_charts.items():
+            # remove the chart key from the row
+            # TODO: is this really the right move? what if the user logs
+            #     a non-custom chart to this key?
```

Comment:
If I understand the flow correctly, this should only occur when the last key logged to k is a custom chart, so it should only ever happen if the key being removed is a custom chart.